pages/plants/[id]/index.js

```diff
@@ -35,7 +35,12 @@ export default function DetailPage({
     mutate,
   } = useSWR(`/api/plants/${id}`, { refreshInterval: 400 });
   if (!session) {
-    return <p>You are not authorized to visit this page.</p>;
+    return (
+      <>
+        <BackArrow link="/overview"></BackArrow>
```

Comment:
This can be a self closing tag if there are no children. 

```suggestion
        <BackArrow link="/overview" />
```